src/ApiConnector.php

```diff
@@ -79,6 +85,9 @@ public static function callServer($url, $data=null, $method='get', $options=arra
 			$request = self::$client->createRequest($method, $url, self::$headers, $data, $options);
 			return $request->send();
 		} catch (\Exception $e) {
+            if (self::$throwExceptions) {
+                throw $e;
+            }

```

Comment:
It seems that the indentation has been fucked up here. We need a linter for this repo too..!